conductr_cli/conduct_info.py

```diff
@@ -18,7 +18,7 @@ def info(args):
     data = [
         {
             'id': ('! ' if bundle.get('hasError', False) else '') + (bundle['bundleId'] if args.long_ids else bundle_utils.short_id(bundle['bundleId'])),
-            'name': bundle['attributes']['bundleName'],
+            'name': bundle_name(args.api_version, bundle['attributes']),
```

Comment:
On reflection I think we should just display the name as-is ie without the compat version. Let's decide seoarelt whether the compat version needs to be displayed...